### PR TITLE
ELEMENTS-541: rely on audit adapter when querying history for a document

### PR DESCRIPTION
**Pull request by guirenard**

### CODE REVIEW

File: nuxeo-audit-page-provider.html [hunk 1]
```diff
@@ -227,26 +237,44 @@
...
+          params.endEventDate = this.params.endDate;
+        }
+        params.currentPageIndex = this.page - 1;
+        params.pageSize = this.pageSize;
+        return this._fetch(this.$.res, params);
+      },
+
+      _fetch: function(exec, params) {
+        if (this._sortKeys.length > 0) {
+          params.sortBy = this._sortKeys.join(",");
```
Comment (afjusto) [inline nuxeo-audit-page-provider.html]: replace double quotes with single

File: nuxeo-audit-page-provider.html [hunk 2]
```diff
@@ -227,26 +237,44 @@
...
-          namedQueryParams: namedQueryParams,
+          providerName: "EVENTS_VIEW",
+          namedQueryParams: this._stringifyJSONObject(this.params),
           currentPageIndex: this.page - 1,
           pageSize: this.pageSize
         };
-
+        return this._fetch(this.$.auditOp, params);
         if (this._sortKeys.length > 0) {
           params.sortBy = this._sortKeys.join(",");
```
Comment (afjusto) [inline nuxeo-audit-page-provider.html]: replace double quotes with single

File: nuxeo-audit-page-provider.html [hunk 3]
```diff
@@ -227,26 +237,44 @@
...
-        var namedQueryParams = this._stringifyJSONObject(this.params);
-        var queryParams = this.docId && this.docId.length > 0 ? this.docId : '';
-        var providerName = this.docId && this.docId.length > 0 ? "DOCUMENT_HISTORY_PROVIDER" : "EVENTS_VIEW";
-
+      _fetchOp: function() {
         var params = {
-          providerName: providerName,
-          queryParams: queryParams,
-          namedQueryParams: namedQueryParams,
+          providerName: "EVENTS_VIEW",
```
Comment (afjusto) [inline nuxeo-audit-page-provider.html]: replace double quotes with single

File: nuxeo-audit-page-provider.html [hunk 4]
```diff
@@ -268,24 +296,22 @@
         }.bind(this));
       },
 
+      get _isForDoc() {
+        return this.docId && this.docId.length > 0;
```
Comment (afjusto) [inline nuxeo-audit-page-provider.html]: `return !!this.docId` should do the same I believe.

**Comment by guirenard** [comment]
:+1:

**Pull request merged**